packages/node_modules/@webex/plugin-meetings/src/config.js

```diff
@@ -45,7 +45,7 @@ export default {
       // TODO: for now this line has to be whitelisted, which is problematic for third party
       clientType: 'TEAMS_CLIENT',
       // Stores the sub client type used when sending metrics
-      subClientType: 'WEB_APP',
```

Comment:
Please add a comment here on why this was changed and submit a JIRA to fix it when the server is fixed.